FavroPlasticExtension/FavroPlasticExtension.csproj

```diff
@@ -13,34 +13,42 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>bin\Debug</OutputPath>
+    <OutputPath>C:\Program Files\PlasticSCM5\client\extensions\favro\</OutputPath>
     <DefineConstants>DEBUG;</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <Optimize>true</Optimize>
-    <OutputPath>bin\Release</OutputPath>
+    <OutputPath>C:\Program Files\PlasticSCM5\client\extensions\favro\</OutputPath>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="issuetrackerinterface">
-      <HintPath>..\lib\issuetrackerinterface.dll</HintPath>

```

Comment:
Don't remove this reference from the project. It makes it impossible to compile it in a machine without plastic installed.

Reply:
I removed the reference because was not able to debug the plugin in a different Plastic version, so I decided to use the actual Plastic installation. I'm going to try another way, I think the problem was that all the dlls was copied into the extensions folder and conflicts with the new version of Plastic. Maybe it's possible to tell the debugger to not deploy specific dll dependencies.

Reply:
I've reverted the changes, but after that I had to update dlls to my current Plastic version because in other case the plugin does not work